website/docs/functions/parse_resource_id.html.markdown

```diff
@@ -66,4 +66,4 @@ normalise_resource_id(id string) string
 
```

Comment:
This file shouldn't be included here, have you rebased this PR recently to ensure changes from `main` are included?